Assets/Scripts/Utility/ImageReader.cs

```diff
@@ -371,6 +371,17 @@ public static ImageData GetImageData(string filename, int record = 0, int frame
 
                     break;
 
+                case ImageTypes.JSON:
+                    string text;
+                    AssetInjection.TextAssetReader.TryRead(filename, out text);
+                    DaggerfallConnect.FallExe.JsonClone jsonClone =
+                        (DaggerfallConnect.FallExe.JsonClone)Game.Serialization.SaveLoadManager.Deserialize(typeof(DaggerfallConnect.FallExe.JsonClone), text);
+                    string sourceName = jsonClone.src;
+                    string actualName = filename.ToLower().Replace(".json", "") + "_" + record + "-" + frame;
```

Comment:
`OrcRace_PaperDollHeadsMale.json` in your mod has a `dest` field, so it seems you intended to put "actual name" in there at first.
Any reason you chose to replace `dest` with `filename.ToLower().Replace(".json", "")`? I guess maybe it's best if the JSON has the same name as the replacement assets. Wondering if you had issues with the `dest` approach

Reply:
No issues, just tried to make life easier for modders😁